### ISSUE TITLE
Need some help working through app logic

### ISSUE THREAD

**Issue by eshimel** [ga-wdi-boston/full-stack-project#43]
Figured this is a separate issue from the registration issue, so I thought I'd be fair and submit a separate issue.  I've written out my app logic and have started trying to group similar events together to write the js functions, but I'd like some guidance on laying all of this out.  I tried to talking it through to my cat this morning with some success, but I'm ready for a knowledgable human.
Below is the outline of data tables and what users need to be able to do with that data.

Stories
 The most recent completed story is available to read on the login/registration page
 Stories are complete when Outline changes
 The story in progress is visible only after a contribution is submitted

Outines 
Outlines are provided from the database upon login
Outlines change every ten Contributions

Clues
Users receive clues from the previous user upon login, unless you are the first user
After they submit their contribution, users submit clues for the next user 

Contributions
Users make a contribution and submit it upon login
A contribution is appended to the story in progress after submission 
Stories are made up of ten contributions

Thanks!

**Comment by gaand** [comment]
Meet me in classroom 1 about 1:30?

**Comment by gaand** [comment]
Covered an approach to specifying big picture in code.  Tentatively closing.  Please reopen if more big picture clarity needed.
